examples_tests/52.SystemTest/main.cpp

```diff
@@ -6,97 +6,240 @@
 #include <iostream>
 #include <cstdio>
 #include <nabla.h>
+// TODO: get all these headers into "nabla.h"
 #include <nbl/ui/CWindowManagerWin32.h>
+#include <nbl/ui/CCursorControlWin32.h>
 #include <nbl/system/ISystem.h>
 #include "../common/CommonAPI.h"
+#include "nbl/system/CStdoutLogger.h"
+#include "nbl/system/CFileLogger.h"
+#include "nbl/system/CColoredStdoutLoggerWin32.h"
```

Comment:
I think I already have moved them?

Reply:
no clue about this, probably @sadiuk  's territory, got in from some merge?

Reply:
not relevant anymore
resolved